From 58fcb8dc7d3ce12a3a87c0b9bb4f8be1cfb4c9bd Mon Sep 17 00:00:00 2001
From: Dirk Chang <dirk@kooiot.com>
Date: Thu, 28 Nov 2024 18:31:50 +0800
Subject: [PATCH 73/89] Revert "dt-bindings: mmc: sdhci-of-dwcmhsc: Add Sophgo
 CV1800B and SG2002 support"

This reverts commit 063fd8bf645fa8baf27c57105c1b53cb1277ee9f.
---
 Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml b/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
index 4d3031d9965f..a43eb837f8da 100644
--- a/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
+++ b/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
@@ -19,9 +19,6 @@ properties:
       - rockchip,rk3568-dwcmshc
       - rockchip,rk3588-dwcmshc
       - snps,dwcmshc-sdhci
-      - sophgo,cv1800b-dwcmshc
-      - sophgo,sg2002-dwcmshc
-      - thead,th1520-dwcmshc
 
   reg:
     maxItems: 1
-- 
2.45.2

